fip-6.md

```diff
@@ -0,0 +1,141 @@
+#### New end point: *deactivate_fio_address* 
+#### New action in new fio.address contract burnaddress
+##### Request
+|Parameter|Required|Format|Definition|
+|---|---|---|---|
+|fio_address|Yes|FIO Address, see FIO Address validation rules.|FIO Address that is being requested to burn.|
+|actor|Yes|FIO account name|FIO account for the signer, the account owning this FIO Address.|
+|max_fee|Yes|max fee SUFs|Maximum amount of SUFs the user is willing to pay for fee. Should be preceded by /get_fee for correct value.|
+|tpid|Yes|FIO Address of TPID, See FIO Address validation rules|FIO Address of the wallet which generates this transaction. This FIO Address will be paid 10% of the fee.See FIO Protocol#TPIDs for details. Set to empty if not known.|
+
+###### Example
+```
+{
+    "fio_address": "purse@alice",
+    "actor": "aftyershcu22",
+    "max_fee": 400000000,
+    "tpid": "rewards@wallet"
+}
+```
+##### Processing
+* Require auth of the actor
+* Request is validated per Exception handling.
+* Removes the fio_address from the fionames table.
+* Removes the TPID entry inside the tpid index table. 
+* Return status json containing status (ok), and fee charged.
+##### Response
+##### Exception handling
+|Error condition|Trigger|Type|fields:name|fields:value|Error message|
+|---|---|---|---|---|---|
+|Invalid FIO Address format|FIO Address format is not valid|400|"fio_address"|Value sent in, e.g. "purse@alice"|"Invalid FIO Address"|
+|Not owner of FIO Address|The signer does not own the address|403||||
+|Invalid fee value|max_fee format is not valid|400|"max_fee"|Value sent in, e.g. "-100"|"Invalid fee value"|
+|Insufficient funds to cover fee|Account does not have enough funds to cover fee|400|"max_fee"|Value sent in, e.g. "400000000"|"Insufficient funds to cover fee"|
+|Fee exceeds maximum|Actual fee is greater than supplied max_fee|400|max_fee"|Value sent in, e.g. "400000000"|"Fee exceeds supplied maximum"|
+|Invalid TPID|TPID is not empty or contains invalid FIO address|400|"tpid"|Value sent in is not empty and not a valid FIO Address format|"TPID must be empty or valid FIO address"|
+|FIO Address not registered|FIO Address is not registered|400|"fio_address"|Value sent in, e.g. "purse@alice"|"FIO Address not registered"|
+|FIO Address is active producer|Supplied FIO Address is registered as producer and is_active = 1|400|"fio_address"|Value sent in, e.g. "purse@alice"|"FIO Address is active producer. Unregister first."|
+|FIO Address is proxy|Supplied FIO Address is registered as proxy|400|"fio_address"|Value sent in, e.g. "purse@alice"|"FIO Address is proxy. Unregister first."|
+
+##### Response
+|Parameter|Format|Definition|
+|---|---|---|
+|status|String|Ok|
+|fee_collected|String|fee amount collected SUFs|
+###### Example
+```
+{
+    "status": "Ok",
+    "fee_collected": "400000000"		
+}
+```
+###### New fee
+A new fee will be created for `deactivate_fio_address` . This fee will be of type 0 and should cost 400000000 SUF.
```

Comment:
Is it bundle eligible? What's fee type 0?

Reply:
Took this style from Eds FIPs but I can be more descriptive here.